config.yml

```diff
@@ -79,11 +106,13 @@ training_pipeline:
     project_rel_path: "model"
 
 logging:
-  loglevel_own: INFO  # override loglevel for packages defined in `own_packages`
-  own_packages: ["__main__", "objectherkenning_openbare_ruimte", "convert_annotations"]
+  loglevel_own: DEBUG  # override loglevel for packages defined in `own_packages`
+  own_packages: ["__main__", "objectherkenning_openbare_ruimte", "convert_annotations",
```

Comment:
we don't need all of the extra own_packages here, we added them in BaaS config.yml because of the @component from AML which was not working well with the logger. As long as under components/ here we do not use AML components, we can simply use the getLogger(__name__)